buddyboss/buddyboss-platform
# URL preview for posts and pages (especially homepage) is inconsistent

### All Submissions:






### Changes proposed in this Pull Request:

Fixes #733  .

### How to test the changes in this Pull Request:

1. Go to the Activity page
2. Share a URL of a blog post or homepage
3. do not enter space or write anything after entering the URL, let the preview load
4. See preview.

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
